### ISSUE TITLE
post_box provided by Deutsche Post have an non-canonical operator preset 

### ISSUE THREAD

**Issue by haukepribnow** [osmlab/name-suggestion-index#6988]
### Current behavior

If you create a post_box operated by Deutsche Post with iD, iD sets the following preset values:

- ```amenity=post_box```
- ```operator=Deutsche Post```
- ```operator:wikidata=Q157645```

### What is wrong?

Canonically, ```operator``` should match the one specified in the associated wikidata. So here, ```operator``` should be "Deutsche Post AG".

### Expected behavior

iD should set the following preset values:

- ```amenity=post_box```
- ```operator=Deutsche Post AG```
- ```operator:wikidata=Q157645```

### Other information

See a discussion about this topic (in German) here: https://www.openstreetmap.org/changeset/125505978

**Comment by bhousel** [comment]
Oh yeah - I think most communities prefer to not include the corporation type in this value, but I have noticed that some Germans like to see the "AG" there. 

The latest planet file that we ingested 3 days ago collected these counts for the different values.
https://raw.githubusercontent.com/ideditor/nsi-collector/main/dist/osm/operators_all.json

<img width="396" alt="Screen Shot 2022-09-01 at 10 02 29 AM" src="https://user-images.githubusercontent.com/38784/187933469-a0e27a3c-2bbb-45f7-b43c-3738a0680c35.png">

So it looks like mappers prefer the version _without_ the "AG" about 3:1.. 

I took a look at the OSM Wikipedia page mentioned in that changeset discussion. It looks like the recommendation to switch from "Deutsche Post" to "Deutsche Post AG" was only changed last year:  https://wiki.openstreetmap.org/w/index.php?title=Deutsche_Postdienstleister&type=revision&diff=2118849&oldid=2103550

So that would explain why lots of post boxes were already mapped without the "AG".  The comment on that change really doesn't link to any other discussion about it, so it feels kind of just like one person editing the wiki to suit their preference.

I guess I'd like to see someone from the German community raise this on a more appropriate forum before we make the change in NSI..  A change like this will trigger validation issues on 27,000 post boxes, and sure we can do it, but I'd like some extra confirmation to make sure that someone else isn't going to show up tomorrow and ask us to change them back.

**Comment by haukepribnow** [comment]
Understood. To be honest, I'm a very low frequency editor who's only doing very limited OSM editing as a hobby. So I wouldn't be the right one to trigger such a discussion.

However, independently from what "local preference" is about this topic, there is another thing to consider:

If you have a ```wikidata``` field, iD does not allow you to change the associated names that are implied by the Wikidata item:
![image](https://user-images.githubusercontent.com/811850/187966020-a1d9cc29-7e35-41d5-a80e-48b0abc206d4.png)

As the [Wikidata item in question](https://www.wikidata.org/wiki/Q157645) has the title "Deutsche Post **AG**", it would only be consequential to reflect the **AG** part also in the ```operator``` field as a preset.

Otherwise Wikidata and OSM would deviate, i.e. the message "you cannot edit the name because it's coming from Wikidata" would be pretty much wrong.

**Comment by bhousel** [comment]
> As the [Wikidata item in question](https://www.wikidata.org/wiki/Q157645) has the title "Deutsche Post **AG**", it would only be consequential to reflect the **AG** part also in the `operator` field as a preset.

Yeah but that doesn't matter very much - anyone can change what the label says on Wikidata.

> If you have a wikidata field, iD does not allow you to change the associated names that are implied by the Wikidata item:

It's true that these fields become locked in iD, but experienced editors can just change the values in the raw tag editor, or by using another editor like JOSM or level0.  The lock is really only intended to signal to casual mappers that they should probably leave the value alone unless they have a really good reason to change it.

**Comment by fabi003** [comment]
The correct operator is really "Deutsche Post AG", with the brand "Deutsche Post" (name from the former state-operated postal service, which is still used as a brand).

There may be a few cases, where the operator really changed, as "Wall AG" (company can go to the stock exchange) back to "Wall GmbH" (normal German capital company) for amenity=toilets in Berlin.

**Comment by bhousel** [comment]
> The correct operator is really "Deutsche Post AG", with the brand "Deutsche Post" (name from the former state-operated postal service, which is still used as a brand).

I think you might be mistaken - I haven't actually looked at this issue in a while, but I do know that in the past people have asked us to _remove_ the "AG" and "GmbH" from the names, and it still looks like today the forms without these things are strongly preferred.

from https://raw.githubusercontent.com/ideditor/nsi-collector/main/dist/osm/operators_all.json
```
  "amenity/post_box|Deutsche Post": 45869,
  "amenity/post_box|Deutsche Post AG": 12704,
```

(Also I remember people requested us to remove these things from the various Deutsche Telekom brands - see https://github.com/osmlab/name-suggestion-index/pull/6199 )

**Comment by fabi003** [comment]
The operator is "Deutsche Post AG" (see https://group.dhl.com/en/legal-notice.html) and the brand is "Deutsche Post". So [it](https://nsi.guide/index.html?t=operators&k=amenity&v=post_box&cc=de#deutschepost-225eac) should be:

`amenity=post_box`
`brand=Deutsche Post`
`operator=Deutsche Post AG`
`operator:wikidata=Q157645`

Entries of operator "Deutsche Telekom AG" seems to be OK.
